modules/api/src/main/java/org/eclipse/xpanse/api/ZitadelAuthorizationApi.java

```diff
@@ -0,0 +1,63 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ * SPDX-FileCopyrightText: Huawei Inc.
+ *
+ */
+
+package org.eclipse.xpanse.api;
+
+import io.swagger.v3.oas.annotations.Operation;
+import io.swagger.v3.oas.annotations.Parameter;
+import io.swagger.v3.oas.annotations.tags.Tag;
+import jakarta.servlet.http.HttpServletResponse;
+import java.io.IOException;
+import lombok.extern.slf4j.Slf4j;
+import org.eclipse.xpanse.modules.models.security.constant.TokenResponse;
+import org.eclipse.xpanse.modules.security.zitadel.ZitadelAuthorizationService;
+import org.springframework.context.annotation.Profile;
+import org.springframework.http.HttpStatus;
+import org.springframework.web.bind.annotation.CrossOrigin;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.ResponseStatus;
+import org.springframework.web.bind.annotation.RestController;
+
+
+/**
+ * REST interface methods for credential management.
+ */
+@Slf4j
+@RestController
+@Profile("zitadel")
+@CrossOrigin
+public class ZitadelAuthorizationApi {
+
+    private final ZitadelAuthorizationService zitadelAuthorizationService;
+
+    public ZitadelAuthorizationApi(ZitadelAuthorizationService zitadelAuthorizationService) {
+        this.zitadelAuthorizationService = zitadelAuthorizationService;
+    }
+
+
+    @Tag(name = "Auth Management",
+            description = "APIs for user authentication.")
+    @Operation(description = "Get and redirect authorization url for user to authorize.")
+    @ResponseStatus(HttpStatus.OK)
+    @GetMapping("/auth/authorize")
+    void redirectAuthorizeUrl(HttpServletResponse response) throws IOException {
```

Comment:
this method can be called just 'authorize' (that what it does at the end by doing a redirect)

Reply:
fixed.